crates/primitives/src/constants/mod.rs

```diff
@@ -60,11 +60,17 @@ pub const EIP1559_DEFAULT_BASE_FEE_MAX_CHANGE_DENOMINATOR: u64 = 8;
 /// Elasticity multiplier as defined in [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559)
 pub const EIP1559_DEFAULT_ELASTICITY_MULTIPLIER: u64 = 2;
 
+/// Base fee max change denominator for Optimism.
+pub const OP_GOERLI_EIP1559_DEFAULT_BASE_FEE_MAX_CHANGE_DENOMINATOR: u64 = 50;
+
+/// Elasticity multiplier for Optimism.
+pub const OP_GOERLI_EIP1559_DEFAULT_ELASTICITY_MULTIPLIER: u64 = 10;
+
```

Comment:
perhaps we should just get rid of all these OP_*_DEFAULT constants entirely and just allow the optimism() and optimism_goerli() factories to specify them?

Reply:
Yes this seems like the best path forward. @clabby is working on an upstream sync and then we are going to point the pr to upstream reth to start eliciting reviews

Reply:
ok let me nuke this before merging then. will have to do it later tonight.

Reply:
done